### PR TITLE
Validations and models specs

### PR DESCRIPTION
**Pull request by LarryIVC**
## *In this pull request, I did the following:*

- [ ] Created a branch.
- [ ] Added validations for User and Post models.
- [ ] Added unit specs.
- [ ] Check the linters.

The following image shows the results of the tests run.
image

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  it 'Name must be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post_counter must be integer' do
+    subject.post_counter = 'a'
+    expect(subject).to_not be_valid
+  end
```
Comment (codepantha) [inline spec/models/user_spec.rb]: - [x] Well done for testing scenarios with invalid attributes. Please test a scenario where "it is valid with valid attributes" 👍 
- [x] Please write tests for the custom methods such as `return_three_recent_posts`, `update_posts_counter` in your model 🙏

File: spec/models/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  it 'Title must be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title must be less than or equal to 250 characters' do
+    subject.title = 'a' * 251
+    expect(subject).to_not be_valid
+  end
```
Comment (codepantha) [inline spec/models/post_spec.rb]: - [x] Well done for testing scenarios with invalid attributes. Please test a scenario where "it is valid with valid attributes" 👍 
- [x] Please write tests for the custom methods such as `return_five_recent_comments`, `update_comments_counter`, `update_likes_counter` in your model 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @LarryIVC 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Added model specs 📌 
✔️ Tests are green 🟢 
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [x] Please write unit tests for the `Comment` and `Like` model as per requirements 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (LarryIVC) [reply spec/models/user_spec.rb]: Thanks @codepantha, I will implement its changes. 👍

File: spec/models/post_spec.rb (hunk 2 above)
Comment (LarryIVC) [reply spec/models/post_spec.rb]: Thanks @codepantha, I will implement its changes. 👍

**Comment by LarryIVC** [comment]
Here I copy the new image from the output of test.html

image

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @LarryIVC, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Validations and model spec implemented ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LarryIVC** [comment]
Thanks @brainconnect93.

**Pull request merged**